lib/gmp/mini-gmp.c

```diff
@@ -55,7 +55,7 @@ see https://www.gnu.org/licenses/.  */
 #include <float.h>
 #endif
 
-
```

Comment:
What is going on here?

Reply:
That's a removal of a form feed character. My script considers it a trailing whitespace.
Just like any other trailing whitespace (e.g. a tab), you usually can't see it, which in my opinion is bad.
I could adjust the script to allow it, but you probably don't want to see any weird character in code files.

Reply:
First time I (don't) see a form feed in source code :sweat_smile: